cli-toolchain/usage.md

```diff
@@ -419,6 +419,236 @@ Now, when Dev Corp is trying to initialize an Amplify Project, the user can sele
 
 You can take a look at [AWS IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html) and the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-role.html) documentation for more details on IAM role and its usage.<br/>
 
+## Mocking and Testing
+
+It is highly recommended that you have completed the Getting Started section of Amplify setup before using local mocking.
+
+- [JavaScript Getting Started](../js/start.md)
+- [Android Getting Started](../android/start.md)
+- [iOS Getting Started](../ios/start.md)
+
+In order to quickly test and debug without pushing all changes in your project to the cloud, Amplify supports *Local Mocking and Testing* for certain categories including API (AWS AppSync), Storage (Amazon DynamoDB and Amazon S3), and Functions (AWS Lambda). This includes using directives from the GraphQL Transformer, editing & debug resolvers, hot reloading, JWT mocking of authorization checks, and even performing S3 operations such as uploading & downloading content.
```

Comment:
Same as above where if we do not support mocking for Storage with DynamoDB we should just mention S3.

Reply:
Yup we call this out in the Storage section on how it's supported.